app/views/foods/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1 class="title is-flex is-justify-content-center">List of Food</h1>
+
+<div class="container" style="width: 60%;">
+  <div class="container is-flex is-justify-content-end mb-4">
+    <%= link_to "Add Food", new_food_path, class: "button is-info" %>
+  </div>
+  <div class="is-flex mt-4 mb-4">
+    <table class="table is-hoverable is-fullwidth">
+      <thead>
+        <tr>
+          <th class="has-text-dark-light">Food</th>
+          <th class="has-text-dark-light">Measuring Unit</th>
+          <th class="has-text-dark-light">Unit Price</th>
+          <th class="has-text-dark-light">Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @foods.each do |food| %>
+          <tr>
+            <td> <%= food.name %></td>
+            <td> <%= food.measurement_unit %></td>
+            <td> $ <%= food.price %></td>
+            <td><%= link_to 'Delete', food_path(food.id), data: {turbo_method: :delete, turbo_confirm: "Are you sure, you want to delete this food?"}, class: "button has-background-danger-light has-text-danger is-small" %></td>
+          </tr>
+        <% end %>
```

Comment:
## [OPTIONAL]

- [ ] I think it should be better to display a text here saying that there are no food, so that the user know that the list of food is empty. eg: 
![image](https://github.com/fatima-najafi/Recipe_App/assets/65068771/0ae69f27-bcfb-4441-9172-254e2e42fdc8)

Reply:
@andersonlebon, thank you very, very much bro... And we appreciate your feedback!